projects/example-app/src/app/auth/effects/auth.effects.ts

```diff
@@ -41,9 +54,26 @@ export class AuthEffects {
     })
   );
 
+  @Effect()
+  logoutConfirmation$ = this.actions$.pipe(
+    ofType(AuthActionTypes.LogoutConfirmation),
+    exhaustMap(() => {
+      const dialogRef = this.dialog.open<
+        LogoutConfirmationDialogComponent,
+        undefined,
+        LogoutConfirmationDialogResult
+      >(LogoutConfirmationDialogComponent);
+
+      return dialogRef.afterClosed();
+    }),
+    filter(result => result === 'OK'),
```

Comment:
Use `LogoutConfirmationDialogResult.OK` instead of the string 'OK'

Reply:
Thankyou! I'll use string enum instead.